reminder_batch_scheduler/reminder_batch.py

```diff
@@ -12,7 +11,7 @@
 
 def main(wave: int, starting_batch: int, max_cases: int, action_plan_id: uuid.UUID, insert_rules: bool = False,
          trigger_date_time: datetime = None):
-    wave_classifiers = constants.WAVE_CLASSIFIERS[wave]
+    wave_classifiers = {'where_clause': constants.WAVE_CLASSIFIERS[wave]}
```

Comment:
Not sure why this is wrapped in a dictionary, it gets read out of it every place it's used that I could see